cmake/modules/SwiftConfigureSDK.cmake

```diff
@@ -186,6 +186,16 @@ macro(configure_sdk_unix name architectures)
         set(SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING "aarch64")
         set(SWIFT_SDK_ANDROID_ARCH_${arch}_PATH "${SWIFT_ANDROID_NDK_PATH}/platforms/android-${SWIFT_ANDROID_API_LEVEL}/arch-arm64")
         set(SWIFT_SDK_ANDROID_ARCH_${arch}_TRIPLE "aarch64-unknown-linux-android")
+      elseif("${arch}" STREQUAL "i686")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_NDK_TRIPLE "i686-linux-android")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING "i686")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_PATH "${SWIFT_ANDROID_NDK_PATH}/platforms/android-${SWIFT_ANDROID_API_LEVEL}/arch-x86")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_TRIPLE "i686-unknown-linux-android")
+      elseif("${arch}" STREQUAL "x86_64")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_NDK_TRIPLE "x86_64-linux-android")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_ALT_SPELLING "x86_64")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_PATH "${SWIFT_ANDROID_NDK_PATH}/platforms/android-${SWIFT_ANDROID_API_LEVEL}/arch-x86_64")
+        set(SWIFT_SDK_ANDROID_ARCH_${arch}_TRIPLE "x86_64-unknown-linux-android")
```

Comment:
I know that you are trying to match the surrounding code, but no point in the `${arch}` substitution here, just spell it out.

Reply:
Sorry, I didn't get .)

I am building swift with the following architectures: 

`-DSWIFT_SDK_ANDROID_ARCHITECTURES="armv7;aarch64;i686;x86_64"`

Without changes above cmake will raise error:

```
CMake Error at cmake/modules/SwiftConfigureSDK.cmake:190 (message):
  unknown arch for android SDK: i686
```

Or you mean that triples `x86_64-unknown-linux-android` and `i686-unknown-linux-android` is not something expected?

Thank you!

Reply:
I think he means that instead of `SWIFT_SDK_ANDROID_ARCH_${arch}_TRIPLE` you can write `SWIFT_SDK_ANDROID_ARCH_x86_64_TRIPLE`.

Personally, I prefer the `${arch}` in the middle because it is easier for copy/pasting in new architectures, and also because it avoids typos in the name of the variables.